### PR TITLE
Pager tests

### PR DESCRIPTION
**Pull request by martiis**

### CODE REVIEW

File: Pager/Adapters/CountAdapter.php [hunk 1]
```diff
@@ -26,11 +26,15 @@ class CountAdapter implements PagerAdapterInterface
...
      * Constructor.
      *
-     * @param int $count
+     * @param array|\Countable|int $value
      */
-    public function __construct($count)
+    public function __construct($value)
     {
-        $this->count = $count;
+        if (is_array($value)) {
```
Comment (GrandLTU) [inline Pager/Adapters/CountAdapter.php]: There also check for Countable interface not just for array as doc comment says it is valid argument.

File: Tests/Unit/Pager/Adapters/CountAdapterTest.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+namespace ONGR\FilterManagerBundle\Tests\Unit\Pager\Adapters;
+
+use ONGR\FilterManagerBundle\Pager\Adapters\CountAdapter;
+
+class CountAdapterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Check if correct results number is returned.
+     */
+    public function testGetTotalResults()
```
Comment (GrandLTU) [inline Tests/Unit/Pager/Adapters/CountAdapterTest.php]: There should be tests for all possible parameter types(int, array, Countable).

File: Tests/Unit/Twig/PagerExtensionTest.php [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    /**
+     * Returns instance of pager extension.
+     *
+     * @param bool $includeGenerate
+     *
+     * @return PagerExtension
+     */
+    protected function getPager($includeGenerate = true)
+    {
+        /** @var $router RouterInterface|\PHPUnit_Framework_MockObject_MockObject $environment */
```
Comment (GrandLTU) [inline Tests/Unit/Twig/PagerExtensionTest.php]: should be `@var RouterInterface|\PHPUnit_Framework_MockObject_MockObject $router`.

**Comment by martiis** [comment]
Will be redone after https://github.com/ongr-io/FilterManagerBundle/pull/52 is merged.

**Pull request closed without merging**